### ISSUE TITLE
How can I modify the protobuf request?

### ISSUE THREAD

**Issue by JamesWDGu** [ProxymanApp/Proxyman#1441]
## Description
Hey guys,
I want to know how to modify the protobuf request.
1. when ues the breakpoint feature, the protobuf response cannot be edited.
2. when use the map local feature, the protobuf response will return an invalid data.

**Comment by NghiaTranUIT** [comment]
Hey, currently, Proxyman doesn't support manipulating the Protobuf data. Only Protobuf Previewer is supported 👍

**Comment by JamesWDGu** [comment]
Thanks @NghiaTranUIT , Do you have plans to implement? It's will be very useful for me. 
Or do I need create a feature request issue?
🥹 🥹 🥹 🙏🙏🙏

**Comment by niceta** [comment]
Hi, @NghiaTranUIT! 

Manipulating protobuf data feature is really important for us, do you have plans to implement it? 🙏 🙏 🙏

**Comment by d2therak** [comment]
I see this has been stale for awhile, wanted to recheck to see if this was going to be a feature in future.

**Comment by NghiaTranUIT** [comment]
@niceta @d2therak May I ask what tool you're going to use to modify the protobuf request? Breakpoint, Map Local or Scripting ?

and do you have the the `desc` file for parsing the protobuf?

**Comment by niceta** [comment]
hi, @NghiaTranUIT !
breakpoint for me is a perfect solution
and yes, I do have desc file for my protobufs

**Comment by amfathi** [comment]
Hi @NghiaTranUIT,
I'm also following up on this. We would love to have the ability to edit the protobufs requests and responses.
We have the proto *.desc that we use to previews.
Thanks 🙏🏻

**Comment by NghiaTranUIT** [comment]
Sorry @amfathi, it's hard to edit the protobuf, so we decided to not support the editing feature. Only Protobuf Previewer is supported 👍

**Comment by siddarthkalra** [comment]
Just want to bump this in-case you ever decide to support. This would be a very useful feature!

**Comment by mfijan-super** [comment]
Hello, any update on this feature?

**Comment by NghiaTranUIT** [comment]
@emanujel not yet because it's really difficult to implement it

**Comment by RenatQAE** [comment]
@NghiaTranUIT 
Hello, can i up this theme again 
Maybe you have already found a solution to this problem?

**Comment by NghiaTranUIT** [comment]
@RenatQAE no, not yet. It's really hard to encode the protobuf on the fly
